src/unit_tests/wazuh_modules/vulnerability_detector/mocks_wm_vuln_detector.c

```diff
@@ -44,6 +44,14 @@ int __wrap_OS_ReadXML(const char *file, OS_XML *_lxml) {
     return mock();
 }
 
+char* __wrap_OS_GetOneContentforElement(OS_XML *_lxml, const char **element_name) {
```

Comment:
We can avoid the compiler warnings with the attribute **unused** for the wrapper parameters.

Reply:
Fixed